src/tarExporter.ts

```diff
@@ -43,7 +43,7 @@ async function saveToTar(fromdir: string, tmpdir: string, toPath: string, repoTa
 	];
 	await fs.writeFile(path.join(tardir, "manifest.json"), JSON.stringify(simpleManifest));
 	await fs.writeFile(path.join(tardir, "config.json"), JSON.stringify(config));
-	await tar.c(
+	await tar.create(
```

Comment:
suggestion: From what I understand `c` is just an alias for `create` which describes to operation better imo. `tar.c(...)` was confusing for me to begin with at least.